src/composables/useDatoNuxtRoute.ts

```diff
@@ -51,6 +52,9 @@ export function useDatoNuxtRoute(page: Page) {
     case 'PersonRecord': {
       return { name: 'language-team-slug', params: { ...sharedParams, slug: page.slug } }
     }
+    case 'MeetRecord': {
+      return { name: 'language-meet-slug', params: { ...sharedParams, slug: page.parents?.[0].slug } }
```

Comment:
```suggestion
      return { name: 'language-meet-slug', params: { ...sharedParams, slug: page.parents[0].slug } }
```

Could this work? Think we'd want to hard fail on a missing parent?